src/madhavi_Raut/MathUsingWhileLoop.java

```diff
@@ -0,0 +1,95 @@
+/*
+Assignment 08
+Write only one program having following methods using While Loop. [4 methods in same program]
+
+1. On user define range print all even numbers.
+Hint : pass start and end index as a parameter.
+Input : Range -> 10 to 15
+Output : Even numbers are:
+10
+12
+14
+
+2. On user define range print all numbers which is divisible by 5.
+Hint : pass start and end index as a parameter.
+Input : Range -> 10 to 30
+Output : Divisible by 5, numbers are:
+10
+15
+20
+25
+30
+
+3. On user define range print all numbers which is divisible by 5 and divisible by 3.
+Hint : pass start and end index as a parameter.
+Input : Range -> 5 to 18
+Output : Divisible by 5 & 3, numbers are:
+15
+
+4. On user define range print all numbers which is divisible by 7 or 13.
+Hint : pass start and end index as a parameter.
+Input : Range -> 5 to 40
+Output : Divisible by 7 or 13, numbers are:
+7 is divisible by 7
+13 is divisible by 13
+14 is divisible by 7
+21 is divisible by 7
+26 is divisible by 13
+28 is divisible by 7
+35 is divisible by 7
+39 is divisible by 13
+ */
+package madhavi_Raut;
+
+public class MathUsingWhileLoop {
+
+	void displayEvenNumbers(int startIndex, int endIndex) {
+		System.out.println("Even numbers are: ");
+		int index = startIndex; 
```

Comment:
you don’t require to create extra variable, you can handle this without **index**

Reply:
same for all other methods